tflint_linux.sh

```diff
@@ -96,6 +96,12 @@ fi
 
 # Install TFLint if missing
 RELEASE_ZIP_NAME="tflint_linux_amd64.zip"
```

Comment:
You should be able to remove this now ;-)

Reply:
Indeed :) 
✅  done